src/celestia/celx_category.cpp

```diff
@@ -0,0 +1,47 @@
+
+#include <fmt/format.h>
+#include "celx.h"
+#include "celx_internal.h"
+#include "celx_category.h" 
+#include <celengine/category.h>
+
+static int category_tostring(lua_State *l)
+{
+    CelxLua celx(l);
+
+    UserCategory *c = *celx.getThis<UserCategory*>();
+    std::string s = fmt::format("[UserCategory:{0}]", c->name());
```

Comment:
Could you replace it with `fmt::printf`? Just for consistence with other code. If `fmt::format` has advantages over `fmt::printf` then we'll allow its usage, but currently let's stick with a single API.